plugins/webp-uploads/picture-element.php

```diff
@@ -168,3 +168,26 @@ function webp_uploads_wrap_image_in_picture( string $image, string $context, int
 		$image
 	);
 }
+
+/**
+ * Adds CSS to make Lightbox trigger BUTTON visible on an Image lightbox when Picture Element is enabled.
+ *
+ * @since n.e.x.t
+ */
+function webp_make_lightbox_trigger_button_visible(): void {
+	if ( has_filter( 'render_block_core/image', 'block_core_image_render_lightbox' ) ) {
+		echo '<style> .wp-lightbox-container > picture + button { opacity: 1 } </style>';
+	}
+}
+
+/**
+ * Add an action to wp_footer conditionally for lighbox trigger button visibility.
+ *
+ * @since n.e.x.t
+ */
+function webp_add_footer_action_for_lightbox_button_visibility(): void {
+	if ( webp_uploads_is_picture_element_enabled() ) {
+		add_action( 'wp_footer', 'webp_make_lightbox_trigger_button_visible' );
+	}
+}
+add_action( 'init', 'webp_add_footer_action_for_lightbox_button_visibility' );
```

Comment:
Do we really need two functions for this? I think it would be fine to unconditionally add the `webp_make_lightbox_trigger_button_visible()` function to `wp_footer`, and simply inside of it include an early return if `! webp_uploads_is_picture_element_enabled()`.

A related question is: Should this be in `wp_footer` or `wp_head`?

Reply:
For classic themes this needs to be in `wp_footer` as otherwise we won't know if there is an Image block using a lightbox.

Reply:
Makes sense, let's stick with `wp_footer` then. My other feedback is still applicable I think.